io/src/lib.rs

```diff
@@ -3,9 +3,9 @@
 use gstd::{prelude::*, ActorId};
 use primitive_types::U256;
 
-/// Escrow wallet ID.
```

Comment:
I think it's unnecessary since it's obvious and documented in other places, e.g.:
https://github.com/gear-dapps/escrow/blob/9c17952096e15ac90e0165eb5f8e64438fc19577/io/src/lib.rs#L43-L46
https://github.com/gear-dapps/escrow/blob/9c17952096e15ac90e0165eb5f8e64438fc19577/io/src/lib.rs#L58-L61
https://github.com/gear-dapps/escrow/blob/9c17952096e15ac90e0165eb5f8e64438fc19577/io/src/lib.rs#L94-L113

Reply:
Okay, if all entities should have a short description, then I'll return it back.